### PR TITLE
fix white scan error

### PR DESCRIPTION
**Pull request by victorwangstone**
- fix linksdk_demo cannot use file system error
- fix whitescan error

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : c3859ca16c4f019133a593017c055163de14e326

**Comment by skylarCai** [comment]
auto build success,number:149

**Comment by skylarCai** [comment]
autotest fail,number:1362

**Review by wuxn9999 — COMMENTED** [review]
https://github.com/alibaba/AliOS-Things/pull/1682

File: components/linksdk/core/aiot_mqtt_api.c [hunk 1]
```diff
@@ -1507,8 +1507,8 @@ static int32_t _core_mqtt_subunsub(core_mqtt_handle_t *mqtt_handle, char *topic,
     pkt[idx++] = (uint8_t)((packet_id >> 8) & 0x00FF);
     pkt[idx++] = (uint8_t)((packet_id) & 0x00FF);
 
-    if (_core_mqtt_5_feature_is_enabled(mqtt_handle)) {
-        _core_write_general_prop(pkt, &idx, general_property, property_len_array, property_len_offset);
+    if (_core_mqtt_5_feature_is_enabled(mqtt_handle) && NULL != general_property) {
```
Comment (skylarCai) [inline components/linksdk/core/aiot_mqtt_api.c:1510]: duplicate

File: solutions/linksdk_demo/package.yaml [hunk 2]
```diff
@@ -35,7 +35,8 @@ supported_boards:
 ## 第五部分：配置信息
 #def_config:                              # 组件的可配置项
 def_config:
-    CLI_IOBOX_ENABLE: 1        # 使能文件系统
+    CLI_IOBOX_ENABLE: 1                   # 使能文件系统
+    VFS_CONFIG_DEVICE_NODES: 256
```
Comment (skylarCai) [inline solutions/linksdk_demo/package.yaml:39]: duplicate

**Review by skylarCai — COMMENTED** [review]
https://github.com/alibaba/AliOS-Things/pull/1694 duplicate

**Pull request closed without merging**